$NetBSD: patch-Makefile,v 1.2 2025/02/12 10:45:28 wiz Exp $

Use explicit rule for building objects; avoid breakage with
different defaults in the share/mk versions.

--- Makefile.orig	2025-02-12 10:43:30.246112733 +0000
+++ Makefile
@@ -23,3 +23,6 @@ install:
 
 clean:
 	rm -f *.o */*.c~ */*.h~ gtk-send-pr gtk-send-pr.core
+
+.c.o:
+	${COMPILE.c} ${.IMPSRC}
